internal/oauthtoken/fake.go

```diff
@@ -28,6 +28,7 @@ import (
 type FakeTokenStore struct {
 	Tokens                       map[string]*oauth2.Token
 	LoadErr, StoreErr, DeleteErr error
+	PanicValue                   any
```

Comment:
Added comments.

For added context that shouldn't be in the fake: the real version can hang indefinitely, so I wanted to test that it's not called. I'd rather not simulate the hang-forever behavior in a test, so it seemed better to panic loudly.

`panic` accepts an `any` rather than an `error`, so that's the value I used. `any` is a builtin type alias for `interface{}`.